test/spec/token.js

```diff
@@ -58,5 +58,159 @@ define(function(require) {
         done();
       });
     });
+
+    it('returns access_token using sessionToken', function (done) {
+      return oauthUtil.setupFrame({
+        oktaAuthArgs: {
+          url: 'https://lboyette.trexcloud.com',
+          clientId: 'NPSfOkH5eZrTy8PMDlvx',
+          redirectUri: 'https://lboyette.trexcloud.com/redirect'
+        },
+        getWithoutPromptArgs: {
+          responseType: 'token',
+          sessionToken: 'testSessionToken'
+        },
+        postMessageSrc: {
+          baseUri: 'https://lboyette.trexcloud.com/oauth2/v1/authorize',
+          queryParams: {
+            'client_id': 'NPSfOkH5eZrTy8PMDlvx',
+            'redirect_uri': 'https://lboyette.trexcloud.com/redirect',
+            'response_type': 'token',
+            'response_mode': 'okta_post_message',
+            'state': oauthUtil.mockedState,
+            'nonce': oauthUtil.mockedNonce,
+            'scope': 'openid email',
+            'prompt': 'none',
+            'sessionToken': 'testSessionToken'
+          }
+        },
+        time: 1449699929,
+        postMessageResp: {
+          'access_token': tokens.standardAccessToken,
+          'token_type': 'Bearer',
+          'expires_in': 3600,
+          'state': oauthUtil.mockedState
+        },
+        expectedResp: {
+          accessToken: tokens.standardAccessToken,
+          expiresAt: 1449703529,
+          scopes: ['openid', 'email'],
+          tokenType: 'Bearer'
+        }
+      })
+      .fin(function() {
+        done();
+      });
+    });
+
+    it('returns access_token and id_token using an array of responseTypes and a sessionToken', function (done) {
+      return oauthUtil.setupFrame({
+        oktaAuthArgs: {
+          url: 'https://lboyette.trexcloud.com',
+          clientId: 'NPSfOkH5eZrTy8PMDlvx',
+          redirectUri: 'https://lboyette.trexcloud.com/redirect'
+        },
+        getWithoutPromptArgs: {
+          responseType: ['id_token', 'token'],
```

Comment:
Since we have the logic, can you also add a test for the ['token', 'id_token'] case, i.e. that we respect the order or arguments passed in the array?